.github/pull_request_template.md

```diff
@@ -1,10 +1,22 @@
-**Related Issue(s)**:  ...
+### Proposed Changes
+ <!--- In case of a bug: Describe what caused the issue and how you solved it-->
 
-**Proposed changes**:
-- ...
+### Related Issues
+- fixes #issue-number
+
+### How Did You Test It?
+<!-- unit tests, integration tests, manual verification, instructions for manual tests -->
+
+### Notes for the Reviewer
+<!-- E.g. point out section where the reviewer  -->
+
+### Screenshots (optional)
+<!-- May be added to illustrate the changes -->
```

Comment:
Personally I think in Haystack is really rare to have screenshots to share. This can only encourage people to post screenshots of stacktraces imho :smile: